pymmcore_plus/core/_mmcore_plus.py

```diff
@@ -447,81 +447,20 @@ def getCameraChannelNames(self) -> Tuple[str, ...]:
     def snapImage(self) -> None:
         return super().snapImage()
 
+    @property
+    def mda(self):
+        return self._mda_engine
+
     def run_mda(self, sequence: MDASequence) -> Thread:

```

Comment:
i know it previously didn't have one, but could you add a very brief docstring here?

Reply:
good call